pkg/controller/azure/database/postgresqlservervirtualnetworkrule/managed.go

```diff
@@ -0,0 +1,175 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package postgresqlservervirtualnetworkrule
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	runtimev1alpha1 "github.com/crossplaneio/crossplane-runtime/apis/core/v1alpha1"
+	"github.com/crossplaneio/crossplane-runtime/pkg/meta"
+	"github.com/crossplaneio/crossplane-runtime/pkg/resource"
+	"github.com/crossplaneio/crossplane/azure/apis/database/v1alpha1"
+	azurev1alpha1 "github.com/crossplaneio/crossplane/azure/apis/v1alpha1"
+	"github.com/crossplaneio/crossplane/pkg/clients/azure"
+)
+
+// Error strings.
+const (
+	errNewClient                                = "cannot create new PostgresqlServerVirtualNetworkRule"
+	errNotPostgresqlServerVirtualNetworkRule    = "managed resource is not an PostgresqlServerVirtualNetworkRule"
+	errCreatePostgresqlServerVirtualNetworkRule = "cannot create PostgresqlServerVirtualNetworkRule"
+	errModifyPostgresqlServerVirtualNetworkRule = "cannot modify PostgresqlServerVirtualNetworkRule"
+	errGetPostgresqlServerVirtualNetworkRule    = "cannot describe PostgresqlServerVirtualNetworkRule"
```

Comment:
"Modify" and "describe" here seem like they might have been copied from the AWS implementation, which uses these words to mean "update" and "get.

Reply:
That is correct. I will update them to be more reflective of the Azure API!